### DISCUSSION TITLE
do not hide error messages

### DISCUSSION THREAD

**Discussion by wnleao** [sevdokimov/log-viewer#43] [Ideas]
![image](https://user-images.githubusercontent.com/2644049/114253990-c02d3e00-9961-11eb-9742-5b9e4fc087bb.png)

I think that some error messages should be visible. For instance, take look at the image, the error message is pretty good to understand the problem, but is not visible in the "failed to read" page. What do you think about showing the complete error message? If you want I could take a look into that.

**Comment by sevdokimov** [comment]
In this case, the error type (status.errorType) is `DirectoryNotVisibleException`. This means that the configuration of the application doesn't allow showing this file. List of available files can be configured in `config.conf` in `logs = [... ]` section. The text in "error" field in the REST response is technical and is not usable for a user. There is a title "Access denied in application configuration" under "access deny text. I'm not a native English speaker, so it would be good if you cleanup my text. The title is defined in `log-viewer/src/main/frontend/src/app/log-view/file-status/file-status.template.html:14` . Please note that there may be a lot of files in multifile mode, that's why I hide detailed error messages in the title.

**Reply by wnleao** [reply]
Okay, I agree with you that the message is a bit technical. I think in the future we could add more logging and use something like spring to work with different log levels --debug, --info, etc. I must be honest that I haven't seen if you've turned that on or not. Don't worry, when I get then I18N issue I will take a better look at the messages.
